requirements/edx-sandbox/post.txt

```diff
@@ -6,5 +6,6 @@
 
 # Packages to install in the Python sandbox for secured execution.
 scipy==0.14.0
-lxml==3.4.4
+# lxml is also in requirements/edx/base.txt
+lxml==3.8.0
```

Comment:
Considering that both lxml and scipy are in the base requirements, can we get rid of both of these lines?